oci/private/descriptor.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+set -o pipefail -o errexit -o nounset
+
+# shellcheck disable=SC2153
+PATH="$HPATH:$PATH"
```

Comment:
What happens today if you replace PATH instead of prepend to it? Maybe add a TODO to be fully hermetic path, if you can't change it yet

Reply:
we need bash toolchain to be fully hermetic, which i will do at once in a follow-up.